wynnset/tapestry-wp
# Add ability to filter nodes based on certain attributes

Closes #478

**General Filtering**

- [x] Add filter component that computes filters based on current window URL
- [x] Should have two dropdowns: one for selecting which attribute to filter on, and the other the attribute value
- [x] On the D3 side, watch for changes in the window URL and update viewable nodes based on this filter

**Filtering by Author**

- [x] Save author information when node is created

**Testing**

- Create a new Tapestry as an admin
  - Make sure that either your default permissions allow add access for authenticated users or when you're adding a node that node has add access
- Login as another user
- Add some nodes
- Press the search button on the top left corner
- On the second combobox, select between the users
- The nodes not made by the user should have lower opacity (unless they're your nodes)
